packages/manager/apps/public-cloud/src/index.html

```diff
@@ -5,8 +5,11 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     </head>
-    <body class="ovhstack h-100" style="overflow: hidden;">
-        <div data-ng-app="ovhStack" data-ng-strict-di class="h-100 d-flex flex-column overflow-hidden" data-ng-controller="PublicCloudController as $ctrl">
+    <body class="ovh-public-cloud-app h-100" style="overflow: hidden;">
```

Comment:
Maybe this inline style could be removed 
```suggestion
    <body class="ovh-public-cloud-app h-100 overfow-hidden">
```

Reply:
@marie-j Thanks for the suggestions. I will take a closer look if `overflow: hidden` rule is still required.